src/syft/core/node/common/action/get_object_action.py

```diff
@@ -45,10 +47,24 @@ def _object2proto(self) -> GetObjectResponseMessage_PB:
             the other public serialization methods if you wish to serialize an
             object.
         """
+
+        ser = self.obj.serialize()
+        # TODO: Fix this hack
```

Comment:
Yikes - let's put this TODO on the top fo the pile for future work. Thank you for documenting it here with a TODO when you found it.